include/kernel.mk

```diff
@@ -244,11 +244,16 @@ $(call KernelPackage/$(1)/config)
     define Package/kmod-$(1)/install
 		  @for entr in $$(call version_filter,$$(FILES)); do \
 			mod="$$$$$$$${entr#-}"; \
-			if grep -q "$$$$$$$${mod##$(LINUX_DIR)/}" "$(LINUX_DIR)/modules.builtin"; then \
+			if grep -q -e"$$$$$$$$mod" '$(LINUX_DIR)/modules.builtin'; then \
```

Comment:
Erk.  Now that I look at this I see I goofed when splitting this into 3 patches.  Originally there was no transition and thus no need to support both `$(LINUX_DIR)/$module_name` and `$module_name` at the same time.  Now due to the split, during these 3 there is such a transition.

Problem is doing the adjustments to `package/kernel` is a >300KB patch due to effecting so many lines.

Reply:
Ick, adjusting to ensure the commits are correct.  Originally this was 1 commit rather than 4.  Perhaps they should be squashed back when brought into OpenWRT's tree.